README.md

```diff
@@ -6,37 +6,46 @@ To get started, save this code snippet as `app.py` and run the below at the end
 ### Serve an Autoscaled Stable Diffusion Endpoint
 
 ```python
-# !pip install 'git+https://github.com/Lightning-AI/stablediffusion.git@lit'
-# !pip install 'git+https://github.com/Lightning-AI/DiffusionWithAutoscaler.git'
 # !pip install 'git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git'
 # !curl https://raw.githubusercontent.com/Lightning-AI/stablediffusion/lit/configs/stable-diffusion/v1-inference.yaml -o v1-inference.yaml
 import lightning as L
-import os, base64, io, ldm, torch
+import os, base64, io, torch
 from diffusion_with_autoscaler import AutoScaler, BatchText, BatchImage, Text, Image, CustomColdStartProxy
 
 PROXY_URL = "https://ulhcn-01gd3c9epmk5xj2y9a9jrrvgt8.litng-ai-03.litng.ai/api/predict"
 
+class FlashAttentionBuildConfig(L.BuildConfig):
+
+    image = "ghcr.io/gridai/lightning-stable-diffusion:v0.2"
```

Comment:
We should probably also host the Dockerfile in the repo, for those who want to reproduce.

Reply:
I will add a TODO for it, but right now. It doesn't seem to work.